src/components/OrganizationDetailPanel.tsx

```diff
@@ -70,6 +77,9 @@ const OrganizationDetailPanel = ({
 					</Dropdown>
 				</PanelActions>
 			</PanelHeader>
+			<PanelBody>
```

Comment:
We won't always want this, but for now:

```suggestion
			<PanelBody padded={false}>
```